src/status_im/contexts/profile/profiles/view.cljs

```diff
@@ -44,32 +43,32 @@
                                               :linear
                                               50))
 
-(defn- show-check-before-syncing
+(defn- navigate-to-new-to-status
```

Comment:
Feel free to ignore this file. As its already reviewed in https://github.com/status-im/status-mobile/pull/21160